Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1010,7 +1008,7 @@ public static void HandleCI(Session session, params string[] parameters)
             WorldObject testContainer = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
             if (testContainer.ItemCapacity > 1)
             {
-                var loot = LootGenerationFactory.CreateTestContainerObject(session.Player, weenieId);
+                Container loot = LootGenerationFactory.CreateTestContainerObject(session.Player, weenieId);
                 loot.ContainerId = session.Player.Guid.Full;
                 session.Player.AddToInventory(loot);
```

Comment:
Just noticed this -- AddToInventory modifies the player -- therefore it must be run within the player's actions (e.g. add this part to an ActionQueue action, then run that)